configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+                                                                        identifier:poi.identifier];
+               [ACPPlaces processRegionEvent:region forRegionEventType:ACPRegionEventTypeEntry];
+           }
+
+           // add the region to our updated list
+           [updatedRegionsUserIsCurrentlyWithin addObject:poi.identifier];
+       }
+   }
+
+   // update user defaults with the new list
+   [[NSUserDefaults standardUserDefaults] setObject:updatedRegionsUserIsCurrentlyWithin forKey:@"regionsUserIsAlreadyIn"];
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS (Swift)" %}
+
+```swift
+func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+    // ask the Places SDK for the 10 nearest Points of Interest based on the user's location
+    ACPPlaces.getNearbyPoints(ofInterest: locations.last!, limit: 10, callback: { (nearbyPoi) in
+        // nearbyPoi is the 10 nearest POIs based on the location
+        self.handleUpdatedPOIs(nearbyPoi!)
+    }) { (error) in
+        // log the error if we have one
+        print("error: \(error)")
+    }
+}
+
+func handleUpdatedPOIs(_ nearbyPois:[ACPPlacesPoi]) {
+    // get the list of regions we know the user is already within from user defaults
+    let regionsUserIsCurrentlyWithin : [String] = UserDefaults.standard.array(forKey: "regionsUserIsAlreadyIn") as! [String]
+
+    // loop through new nearbyPoi and post entry events for pois that aren't already in our list
+    // also creating the new list of known regions that the user is in
+    var updatedRegionsUserIsCurrentlyWithin: [String] = []
+    for poi in nearbyPois {
+        // check if the user is in this poi
+        if poi.userIsWithin {
+            // the user is in the poi, now we need to make sure we haven't already recorded the entry event
+            if !regionsUserIsCurrentlyWithin.contains(poi.identifier!) {
+                let region = CLCircularRegion.init(center: CLLocationCoordinate2D.init(latitude: poi.latitude, longitude: poi.longitude), radius: CLLocationDistance(poi.radius), identifier: poi.identifier!)
+                ACPPlaces.processRegionEvent(region, for: .entry)
+            }
+
+            // add the region to our updated list
+            updatedRegionsUserIsCurrentlyWithin.append(poi.identifier!)
+        }
+    }
+
+    // update user defaults with the new list
+    UserDefaults.standard.set(updatedRegionsUserIsCurrentlyWithin, forKey: "regionsUserIsAlreadyIn")
+}
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+## Next Steps
+
+Triggering Places entry events in the SDK is just the start. Because of the triggering entry events, all of the data that defines your Points of Interest can be used by the rest of the SDK via `data elements` in Launch. With Launch `rules`, you can dynamically attach the Places data to incoming events processed by the SDK. For example, you can attach all of the meta data of a POI that the user is in and send it to Analytics as context data.
```

Comment:
Please reword to:
"In addition to triggering Places entry events in the SDK, because of the triggering entry events, all of the data that defines your POIs can be used by the rest of the SDK via `data elements` in Launch. With Launch `rules`, you can dynamically attach the Places data to incoming events that are processed by the SDK. For example, you can attach the metadata of the POI in which the user is located and send the data to Analytics as context data. For more information, see [Using Places with other Adobe Solutions](todo:).